pkg/bbgo/serialmarketdatastore.go

```diff
@@ -1,22 +1,34 @@
 package bbgo
 
 import (
+	"context"
+	"sync"
 	"time"
 
+	"github.com/c9s/bbgo/pkg/fixedpoint"
 	"github.com/c9s/bbgo/pkg/types"
 )
 
 type SerialMarketDataStore struct {
 	*MarketDataStore
-	KLines       map[types.Interval]*types.KLine
-	Subscription []types.Interval
+	UseAggTrade              bool
+	KLines                   map[types.Interval]*types.KLine
+	MinInterval              types.Interval
+	Subscription             []types.Interval
+	o, h, l, c, v, qv, price fixedpoint.Value
+	lock                     sync.Mutex
```

Comment:
go convention usually use `mu` to `mutex`

Reply:
done